add all files

Danik2021 opened the pull request.

Danik2021 committed:
add all files

Danik2021 committed:
bugFix

folder img replaced on image, picture have a size, email on next stroke without </br>

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+          </ul>
+        </nav>
+        <div class="header-menu">
+          <a class="email" href="info@devstudio.com">info@devstudio.com</a>
```
Тут було б непогано додати до пошти mailto:

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+        </div>
+      </header>
+      <main>
+        <section class="section-one">
```
Краще не використовувати назви класів типу  "section-one", краще їх називати щоб ти розумів що там. Наприклад "baner", тощо. Просто по іншому ти потім в майбутньому не розберешся що де у тебе на сторінці) уяви ситуацію, коли в тебе таких може бути 100 секцій і тобі треба редагувати стилі з банеру) і ти такий підеш спочатку гортати html шукати який там класс section-4 чи section-10) а маючи більш чітку назву ти просто сразу йдеш в css файл та розумієш що тобі треба редагувати селектор для класу "baner"

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+        </section>
+        <section class="section-two">
+          <div class="cont-section-two">
+            <dl>
```
Це не зовсім той випадок для використання тегів dl, dt, dd, вони більш підходять коли в тебе прям якийсь словник, типу вікіпедії, тобто коли в тебе є якесь поняття та роз'яснення до цього поняття. Типу як словник слів, по аналогії вікіпедії. Взагалі ці теги рідко використовуються в дуже специфічних ситуаціях, хіба коли будеш писати сайт словник)

xrustics replied:
Тут краще використовувати список, у якому кожен блок це буде li який буде містити якийсь h3 і p

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+        <section class="section-three">
+          <div class="cont-section-three">
+            <h2>Чим ми займаємося?</h2>
+            <div class="cont-img-work">
```
Це теж список а не просто в div) 
Коли ти бачиш декілька однотипних елементів - це скоріше за все список)

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+          </div>
+        </section>
+        <section class="section-four">
+          <div class="cont-section-four">
```
І це теж список карточок людей команди, його треба в ul > li. По суті це однотипні елементи. Типу однакові карточки і все відноситься до одного - це список людей команди

xrustics replied:
Після заголовку всі картки то ul > li

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+                <figcaption>Ігор Демʼяненко</figcaption>
+                <p class="specialty-one">Product Designer</p>
+              </figure>
+              <figure class="figure-two">
```
Ну з назвою класів треба щось придумати) бо картинка-1, картинка-2 це буде дуже важко підтримувати. Треба більш чіткі назви, щоб ти розумів що це за картинка по назві.

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+              <figure class="figure-two">
+                <img src="image/frontend_developer.svg" alt="Olaga Repina" />
+                <figcaption>Ольга Рєпіна</figcaption>
+                <p class="specialty-two">Frontend Developer</p>
```
Мені здається тут тобі не потрібно мати декілька класів specialty-one, specialty-two і т.д. Там же стилі однакові для кожної такої фрази, тому можна просто створити один клас типу specialty і його навішати на кожен елемент такої фрази. Таким чином ти перевикористаєш код

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,118 @@
+          <div class="cont-footer">
+            <div class="logo-two">Web<span class="span-logo">Studio</span></div>
+            <address class="location">м. Київ, пр-т Лесі Українки, 26</address>
+            <a class="email-two" href="info@devstudio.com">info@devstudio.com</a>
```
Додай ще mailto:

Danik2021 replied:
✔✔✔

xrustics commented on script.js:
```diff
@@ -0,0 +1,6 @@
+let str = 'lorem ipsum dolor first second third';
```
Цей код десь використовується? Якщо ні то видали зайве

Danik2021 replied:
✔✔✔

xrustics reviewed (changes requested):
Треба переглянути імена для класів та оптимізувати їх в плані коли в тебе однакові елементи для них не потрібно для кожного створювати окремо клас, їм треба просто дати той самий клас. А так в цілому дуже непогано для першого разу)
Бачу ти навіть css написав, але перша домашка там без css можна було)

Danik2021 committed:
bugFix Ruslan

malito add,
section names have changed,
dd, dl, dt have delete,
div have changed on li,
images have new names

xrustics commented on index.html:
```diff
@@ -0,0 +1,134 @@
+          </div>
+        </section>
+        <section class="aboutUs-section">
+          <div class="cont-aboutUs-section">
```
дивись я трохи про інше казав)
ось так треба оформити цей список:
![image](https://github.com/Danik2021/goit-markup-hw-01/assets/59570719/f9fad707-99fd-4592-b93d-715a7c0ba307)

xrustics replied:
тоді і div не требе)

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,134 @@
+        <section class="whatDo-section">
+          <div class="cont-whatDo-section">
+            <h2>Чим ми займаємося?</h2>
+            <ul class="ul-img-work">
```
Ооо! А ось тут супер, зробив правильно 👍

xrustics commented on index.html:
```diff
@@ -0,0 +1,134 @@
+          <div class="cont-ourTeam-section">
+            <h2>Наша команда</h2>
+            <div class="cont-img-team">
+              <ul class="cont-employee">
```
а тут неправильно) У тебе має бути один список `ul` а карточки це елементи списку будуть (li а не ul списки), типу так:
`<li class="cont-employee">
                <img src="image/frontend_developer.svg" alt="Olaga Repina" />
                <h3 class="employee">Ольга Рєпіна</h3>
                <p class="specialty">Frontend Developer</p>
              </li>`

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,134 @@
+  </head>
+
+  <body>
+    <div class="wrapper">
```
подумай, щоб позбутися цього враперу з div, ти ж по факту можеш повісити цей же клас на сам тег body

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,134 @@
+      <footer>
+        <div class="footer">
+          <div class="cont-footer">
+            <div class="logo-two">Web<span class="span-logo">Studio</span></div>
```
"logo-two" - зміни назву на більш точну, типу footer-logo чи якось так. Просто щоб ти розумів по назві що це)

Danik2021 replied:
✔✔✔

Danik2021 committed:
bugFix Ruslan

wrapper delete
section aboutUs has been corrected,
section ourTeam has corrected too,
logo-two changed on footer-logo

xrustics commented on index.html:
```diff
@@ -0,0 +1,130 @@
+    </header>
+    <main>
+      <section class="main-section">
+        <h1>Ефективні рішення для вашого бізнесу</h1>
```
зауваж, що h1 на сторінці має бути лише один!
Хорошою практикою є використання h1 для логотипу сайту в хедері а далі кожен наступний заголовок дивишся по структурі, типу в хедері лого це h1, в банері "Ефективні рішення для вашого бізнесу" тут краще h2, блок  aboutUs - h3 (як ти вже зробив) і так далі

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,130 @@
+    <main>
+      <section class="main-section">
+        <h1>Ефективні рішення для вашого бізнесу</h1>
+        <div class="button">
```
тут семантично правильніше використати за допомогою тегу button

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,130 @@
+      </section>
+    </main>
+    <footer>
+      <div class="footer">
```
давай попробуємо позбутися зайвих div обгорток в футері. Тут же щось можна повісити безпосередньо на тег footer (клас)

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -0,0 +1,130 @@
+
+  <body>
+    <header>
+      <div class="logo">
```
Давай робити одразу як на реальному проекті, лого в хедері як правило це h1 (який має бути один на сторінці), можна зробити так - `<h1><a href="./index.html">...</a></h1>`

Danik2021 replied:
✔✔✔

Danik2021 committed:
bugFix Ruslan

div in logo replaced on h1
excessive div in footer has deleted

xrustics reviewed (approved):
Молодець, все ок 👍
